packages/contentstack/package.json

```diff
@@ -49,8 +46,7 @@
     "winston": "^3.7.2"
   },
   "devDependencies": {
-    "@oclif/dev-cli": "^1.22.2",
-    "@oclif/plugin-help": "^5.1.12",
+    "oclif": "^3",
     "@oclif/test": "^1.2.8",
```

Comment:
Do we still need to install oclif/test separately if we already have oclif installed above?

Reply:
As oclif dev dependency package is different and does not include @oclif/test package we have install @oclif/test package.